packages/sdk/src/hooks/useCredentials.tsx

```diff
@@ -0,0 +1,33 @@
+import { useMemo } from 'react'
+import { useMdocRecords } from '../providers/MdocProvider'
+import { useSdJwtVcRecords } from '../providers/SdJwtVcProvider'
+import { useW3cCredentialRecords } from '../providers/W3cCredentialsProvider'
+
+/**
+ *
+ * @todo add Display information to the records
+ *
+ */
+export const useCredentials = ({
+  removeCanonicalRecords = true,
+}: {
+  removeCanonicalRecords?: boolean
+} = {}) => {
+  const { w3cCredentialRecords, isLoading: isLoadingW3c } = useW3cCredentialRecords()
+  const { sdJwtVcRecords, isLoading: isLoadingSdJwt } = useSdJwtVcRecords()
+  const { mdocRecords, isLoading: isLoadingMdoc } = useMdocRecords()
+
+  const credentials = useMemo(() => {
+    const allRecords = [...w3cCredentialRecords, ...mdocRecords, ...sdJwtVcRecords]
+
+    // Sort by creation date
+    const sortedRecords = allRecords.sort((a, b) => b.createdAt.getTime() - a.createdAt.getTime())
+
+    return sortedRecords
+  }, [w3cCredentialRecords, sdJwtVcRecords, mdocRecords])
+
+  return {
+    credentials,
```

Comment:
you mentioned adding display etc. Will we also abstract away the Credo records, and just return objects with the needed params?

Reply:
Yes, ideally I would like to return the bare minimum.